### ISSUE TITLE
CPU Engine totalTime must be greaterThan0

### ISSUE THREAD

**Issue by saantiaguilera** [Kyson/AndroidGodEye#16]
Showing stacktrace, it appears randomly after using the app for a bit

```
E/AndroidRuntime: FATAL EXCEPTION: RxComputationThreadPool-8
                                                   Process: com.myapp, PID: 29724
                                                   io.reactivex.exceptions.OnErrorNotImplementedException: totalTime must greater than 0
                                                       at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:704)
                                                       at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:701)
                                                       at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:74)
                                                       at io.reactivex.observers.SerializedObserver.onError(SerializedObserver.java:155)
                                                       at io.reactivex.internal.operators.observable.ObservableConcatMap$SourceObserver$InnerObserver.onError(ObservableConcatMap.java:254)
                                                       at io.reactivex.internal.observers.BasicFuseableObserver.onError(BasicFuseableObserver.java:100)
                                                       at io.reactivex.internal.observers.BasicFuseableObserver.fail(BasicFuseableObserver.java:110)
                                                       at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:61)
                                                       at io.reactivex.internal.operators.observable.ObservableTimer$TimerObserver.run(ObservableTimer.java:67)
                                                       at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
                                                       at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
                                                       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
                                                       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
                                                       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
                                                       at java.lang.Thread.run(Thread.java:762)
                                                    Caused by: cn.hikyson.godeye.core.internal.exception.GodEyeInvalidDataException: totalTime must greater than 0
                                                       at cn.hikyson.godeye.core.internal.modules.cpu.CpuEngine$3.apply(CpuEngine.java:61)
                                                       at cn.hikyson.godeye.core.internal.modules.cpu.CpuEngine$3.apply(CpuEngine.java:55)
```

**Comment by Kyson** [comment]
cpu data exception.
i will resolve next version,maybe 1.8.x

**Comment by Kyson** [comment]
@saantiaguilera resolved at versoin 1.7.4, thank you.
